src/components/Dashboard/Dashboard.jsx

```diff
@@ -14,19 +14,24 @@ import SearchBar from "material-ui-search-bar";
 // import LinearProgress from "@material-ui/core/LinearProgress";
 import { useStyleDashboard } from "./Widgets/Style";
 import FiltersFlat from "./Widgets/FiltersFlat";
-import { ProfileContext } from '../../profileContext' 
-
+import { ProfileContext } from "../../profileContext";
+import { db } from "../../firebase";
+import CircularProgress from "@material-ui/core/CircularProgress";
 
 export default function Dashboard() {
-  const [profile, ] = useContext(ProfileContext)
-
+  const [profile] = useContext(ProfileContext);
+  const [posts, setPosts] = useState();
   const [state, setState] = React.useState({
```

Comment:
Why are you using `React.useState` when you have it imported already?

Reply:
Done 783ee049a3396f564058a194d966ebf9d3ca9f7e